drivers/interrupt_controller/intc_intel_vtd.h

```diff
@@ -13,7 +13,7 @@
  * it's always 0
  */
 #define VTD_MSI_MAP(int_idx) \
-	((0x0FEE << 20) | (int_idx << 5) | VTD_INT_SHV | VTD_INT_FORMAT)
+	((0x0FEE << 20) | VTD_INT_SHV | VTD_INT_FORMAT)
```

Comment:
MSI address register needs the interrupt index. Check the VT-D specs section 5.1.5.2

Reply:
Please see my previous comment on same topics